### PR TITLE
Add tour guide label and make QR code a link

### PR DESCRIPTION
**Pull request by hunchr**
Needs: https://github.com/hitobito/hitobito/pull/2649
Fixes: #533

image

### CODE REVIEW

**Comment by hunchr** [comment]
wip: specs

**Review by rnestler — COMMENTED** [review]
Maybe also take a look at `spec/features/membership_verification_spec.rb` if you can extend a spec or add an additional one.

File: spec/fixtures/people.yml [hunk 1]
```diff
@@ -78,3 +78,13 @@ tourenchef:
...
+
+tourenleiter:
+  id: 600_007
+  first_name: Alina
+  last_name: Müller
+  email: alina-müller@hitobito.example.com
+  gender: f
+  primary_group: matterhorn_tourenkommission
+  birthday: <%= Date.new(1993, 6, 12) %>
+  language: de
```
[the remark this answers is not in the document]
Comment (rnestler) [reply spec/fixtures/people.yml:90]: My bad. I told @hunchr that you seem to be using fixtures and to better extend those.

[the remark this answers is not in the document]
Comment (mtnstar) [reply spec/fixtures/people.yml:90]: np, somehow this is the way we're doing it.
one could use the fixture mitglied and then add a tourenleiter rolle to it, even without fabricator

**Pull request merged**